### ISSUE TITLE
Configure GeoFence access rules: It is not possible to filter the rule using the "draw AOI filters"

### ISSUE THREAD

**Issue by ElenaGallo** [geosolutions-it/MapStore2#9964]
## Description
 
From the new rule panel, it is not possible to draw a filter.

## How to reproduce

-  Open the Configure access rules
-  Click on Add new rule
- Add a workspace
- Click on Filter tab
- Draw a filter on map

- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>

(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

## Other useful information

Related to [this cucumber test](https://studio.cucumber.io/projects/47740/test-runs/930180/folder-snapshots/10949546/scenario-snapshots/34294598/test-snapshots/45777529)

**Comment by tdipisa** [comment]
Let's verify and update issue preparing it for development if necessary.

**Comment by tdipisa** [comment]
@rowheat02 can you please verify and try to reproduce this? Probably you can also use our DEV for a first check. Then you have to report the tech approach as well as the related effort.

**Comment by rowheat02** [comment]
@tdipisa 
I tried to follow exactly as here: https://github.com/geosolutions-it/MapStore2/issues/9964#issue-2125518937

<img width="523" height="790" alt="Image" src="https://github.com/user-attachments/assets/fd519dab-67b3-40c8-8c62-71ee57c2d43f" />

The Filter tab is disabled( along with Styles, filter, and attribute rule) 
- Not sure why it is disabled. No Messages displayed. 
- And the Cucumber test is also not accessible.
- Filter tab expected to be disabled? Or did I miss a configuration that will enable the filter tab?

**Comment by tdipisa** [comment]
ok @rowheat02 let's put this on hold as this need a standalone geofence instance to be enabled.

**Comment by tdipisa** [comment]
@rowheat02 I confirm that to enable rule details a standalone geofence must be used. It is also reported by the MS doc [here](https://docs.mapstore.geosolutionsgroup.com/en/master/user-guide/rule-manager/#manage-rules).

<img width="791" height="235" alt="Image" src="https://github.com/user-attachments/assets/5580fc72-f63d-42de-91ec-63a12a01e907" />

@ElenaGallo please can you clarify where you have noticed that problem? Was on a standalone GeoFence or in a GS embebbed one. I think it is the firs one as the issue is explicitly referring to a function of the Filters tab.

If you activate the area of Interest option

<img width="480" height="369" alt="Image" src="https://github.com/user-attachments/assets/1e864d54-d095-4985-9ffa-386c870d398c" />

MS crashes:

<img width="708" height="402" alt="Image" src="https://github.com/user-attachments/assets/2166e156-f3db-4fbf-a180-8d70b9199c83" />


There is also a wrong expand button to be removed. Probably a CSS issue:

<img width="273" height="79" alt="Image" src="https://github.com/user-attachments/assets/9bacf986-004a-4215-bc38-a4b4473976c6" />

as a side note@rowheat02, please remember also that for enabling rule details the layer should be also specified in the rule., of course.

**Comment by ElenaGallo** [comment]
@tdipisa Yes, I was referring to the issue you also described in your comment https://github.com/geosolutions-it/MapStore2/issues/9964#issuecomment-3951196004, when the filter tab is selectable, meaning with a standalone GeoFence.

**Comment by MV88** [comment]
@tdipisa the problem looks like a bad format of numbers to apply the toFixed function

<img width="1901" height="524" alt="Image" src="https://github.com/user-attachments/assets/69774c27-3f0b-41e4-a710-eb6007d86101" />


with this line it is fixed

```js
return Math.abs(Number(a).toFixed(8) - Number(b).toFixed(8)) <= 0.00000001;
```

also this isNearlyEqual  is used in many places, maybe we can make a utility function and reuse it

**Comment by tdipisa** [comment]
@MV88 thank you 
it seems a small bug fix. Can you please quote it in the estimate please?
